assignments/prototype-refactor.js

```diff
@@ -7,3 +7,170 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
+
+/*
+  === GameObject ===
+  * createdAt
+  * dimensions (These represent the character's size in the video game)
+  * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
+*/
+    class GameObject {
+        constructor(charAtt){
+            this.createdAt = charAtt.createdAt;
```

Comment:
Since we know that createdAt attribute is an object, does it need () for us to invoke it, or not?
If it does, i want you to ask yourself why...
if it doesn't, than try to look into it, how come some functions need () to get invoked, and some don't? Why do we put parenthesis when we call our functions?